packages/create-sitecore-jss/src/common/prompts/styleguide.ts

```diff
@@ -0,0 +1,31 @@
+import { Answers, DistinctQuestion } from 'inquirer';
+import chalk from 'chalk';
+
+export interface StyleguideAnswer extends Answers {
+  language?: string;
+}
+
+const LANGUAGE_REGEXP = /^(([a-z]{2}-[A-Z]{2})|([a-z]{2}))$/;
+
+export const styleguidePrompts: DistinctQuestion<StyleguideAnswer>[] = [
+  {
+    type: 'input',
+    name: 'language',
+    message:
+      'Which additional language do you want to support (en is default and required)? Leave empty if not needed',
+    validate: (input: string): boolean => {
```

Comment:
Handle case where user inputs en (this will cause duplicate filenames and break things)